armi/bookkeeping/tests/test_snapshot.py

```diff
@@ -27,7 +41,23 @@ def setUpClass(self):
 
     def setUp(self):
         self.cs.revertToDefaults()
-        self.si = snapshotInterface.SnapshotInterface(None, self.cs)
+        self.si = snapshotInterface.SnapshotInterface(MockReactor(self.cs), self.cs)
+
+    @patch("armi.operators.operator.Operator.snapshotRequest")
+    def test_interactEveryNode(self, mockSnapshotRequest):
```

Comment:
This is really interesting. You didn't implement `mockSnapshotRequest`.  Normally, people have to mock a return value at least.  It's cool that it works though. Nice.